INSTALLING.md

```diff
@@ -157,6 +157,25 @@ sudo apt-get install cpufrequtils
 sudo cpupower frequency-set -g performance
 ```
 
+#### Option 3: From Docker Release Image
+
+Download the latest Docker release from our
```

Comment:
There is no docker release in the releases link.

It might be better to just mention we provide docker images for each release because the actual release page does not give any mention of docker.

A hyperlink like the following might work better here
https://github.com/orgs/tenstorrent/packages?tab=packages&q=tt-metalium+release

Reply:
What do you think if I add a link to the Release Notes that are published with the releases?

Reply:
I switched the link to point to the correct images.